recipe/meta.yaml

```diff
@@ -97,6 +97,8 @@ test:
     - make                       # [not win]
     # Need zlib to compile test programs
     - zlib
+    # ensure new libgfortran is co-installable
```

Comment:
```suggestion
    # In 2025/07/23 we had issues with co-installation of libgfortran
    # This added requirement may not age well with time and may be removed
    # in the not so distant future
    # Quoting h-vetinari in https://github.com/conda-forge/hdf5-feedstock/pull/267#issue-3256582023
    # There was an issue with the new gfortran compilers, see https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/1054. 
    # This might be the reasons for issues like in https://github.com/conda-forge/conduit-feedstock/pull/52 that look like
    #
    # ensure new libgfortran is co-installable
```

Reply:
What makes you think that this requirement will not age well?

Reply:
at some point the lower bound will be implicitly set to be higher.

may not be useful to have this added line of code.

Reply:
> at some point the lower bound will be implicitly set to be higher.

Sure libgfortran will have higher and higher versions. But that's not what this is about, which is that some combination of conda-build and/or the repodata packages hallucinate a dependency `libgfortran =5`, which is what this test guards against.

> may not be useful to have this added line of code.

Given that this is a problem occurring today in other feedstocks, this statement is incorrect on its face. Obviously the benefit is in avoiding those failures that are currently being observed, and guard against future such regressions.

Reply:
Can we revert https://github.com/conda-forge/hdf5-feedstock/pull/267/commits/54e97e506b63fb3fa40e227122a6c4cddf6d4436 or come up with a formulation that we both agree with?

Reply:
Sure. Feel free to to force push